File: java/regexExample.java
import java.util.regex.Matcher;
import java.util.regex.Pattern;

public class regexExample {
    public static void main(String[] args) {
	String line = "This order was placed for QT3000! OK?";
	String regex = "(.*)(\\d+)(.*)";

	// Create a Pattern Object
	Pattern pattern = Pattern.compile(regex);

	// Create a Matcher Object
	Matcher match = pattern.matcher(line);

	if(match.find()){
	    System.out.println("Found value: "+match.group(0)+"\nFound value: "+match.group(1)+"\nFound value: "+match.group(2));
	}
	else{
	    System.out.println("Not found!!!!");
	}
    }
}

File: python/regex-example.py
from re import findall


string = "I am sTux, Born April 7th 2005"


for regex in [r"[0-9]", r"[0-9]+", r"[a-z]", r"[A-Z]", r"[a-z]+", r"[A-Z]+", r"[a-zA-z]", r"[a-zA-Z]+",r"[a-zA-Z0-9]", r"[a-zA-Z0-9]+"]:
    print(f"Launching regex {regex}")
    print(f"Found {findall(string=string, pattern=regex)} from {string}")

